repairnator/repairnator-realtime/src/main/java/fr/inria/spirals/repairnator/realtime/InspectProcessDuration.java

```diff
@@ -12,23 +14,29 @@
 public class InspectProcessDuration implements Runnable {
     private static final Logger LOGGER = LoggerFactory.getLogger(InspectProcessDuration.class);
     private static final int SLEEP_TIME = 10; // seconds
-
+    private PIPELINE_MODE pipelineMode = PIPELINE_MODE.NOPE;
```

Comment:
there is no point in having this here, this is part of the global config, remove

Reply:
Yes, this can be removed.